receiver/awsxrayreceiver/README.md

```diff
@@ -0,0 +1,68 @@
+# AWS X-Ray Receiver
+
+**Status: alpha**
+
+## Overview
+The AWS X-Ray receiver accepts segments (i.e. spans) in the [X-Ray Segment format](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html).
+This enables the collector to receive spans emitted by the existing X-Ray SDK. [Centralized sampling](https://github.com/aws/aws-xray-daemon/blob/master/CHANGELOG.md#300-2018-08-28) is also supported via a local TCP port.
+
+The requests sent to AWS are authenticated using the mechanism documented [here](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+
+## Configuration
+
+Example:
+
+```yaml
+receivers:
+  aws_xray:
+    endpoint: localhost:2000
+    proxy_server:
+      tcp_endpoint: localhost:2000
+      proxy_address: ""
+      insecure: false
```

Comment:
Isn't this

```
tls_setting:
  insecure: false
  server_name_override: ""
```
?

Reply:
No, the field is defined as:
```
TLSSetting configtls.TLSClientSetting `mapstructure:",squash"`
```
which is similar to how the other plugin define it. And based on:
* https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/receiver/carbonreceiver/config.go#L34 (`configmodels.ReceiverSettings` has a nested field `endpoint`)
* https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/receiver/carbonreceiver/testdata/config.yaml#L6

when a field is designated as `mapstructure:",squash"`, there will be no nested structure.

Reply:
Ah got it - thanks! LGTM